src/main/java/sirius/pasta/tagliatelle/compiler/TemplateCompiler.java

```diff
@@ -417,6 +425,25 @@ private void parseAttributes(TagHandler handler) {
             }
             consumeExpectedCharacter('"');
         }
+
+        checkMissingAttributes(handler, attributes);
+    }
+
+    /**
+     * Checks if all required attributes are present.
+     *
+     * @param handler    the tag handler
+     * @param attributes the set of attributes which were parsed
+     */
+    private void checkMissingAttributes(TagHandler handler, Set<String> attributes) {
+        Set<String> missingAttributes = new HashSet<>(handler.getRequiredAttributeNames());
+        missingAttributes.removeAll(attributes);
+        if (!missingAttributes.isEmpty()) {
+            missingAttributes.forEach(attr -> context.error(reader.current(),
+                                                            "Missing required attribute. %s missing the required attribute '%s'.",
+                                                            handler.getTagName(),
+                                                            attr));
```

Comment:
Style: würd mir hier die mutability sparen und direkt nen stream drüber laufen lassen: 
So ungefähr: requiredAttribute.stream.filter(predicate.not(foundAttributes::contains).foreach...